src/main/pg/gyrodev.c

```diff
@@ -78,8 +95,8 @@
 
 #ifndef GYRO_1_CUSTOM_ALIGN
 #define GYRO_1_CUSTOM_ALIGN     SENSOR_ALIGNMENT_FROM_STD(GYRO_1_ALIGN)
-#else
-STATIC_ASSERT(GYRO_1_ALIGN == ALIGN_CUSTOM, "GYRO_1_ALIGN and GYRO_1_CUSTOM_ALIGN mixed");
+// #else
```

Comment:
https://github.com/betaflight/config/pull/643

Reply:
Thanks, a lot of configurations use a mix of alignment macros :confused:

Reply:
Still a lot of configurations in config repository mix both alignment method.

Reply:
- As custom gyro defines do not work since fw 4.5 (as most were migrated from unified targets) should we continue to remove custom `GYRO_[x]_ALIGN_[RPY]` macro if `GYRO_[x]_ALIGN` is defined  ?
- Why were they mixed before in unified targets ?

Reply:
259 configs affected ... 
Even than, having identical information in two places is bad idea, it will eventually  lead to problems.

Reply:
- As setting are mutual exclusive could we just use RPY only and remove custom alignment to simplify configuration ?
- This would mimic board alignment as we can use define `USE_GYRO_1_ALIGN SENSOR_ALIGNMENT( R, P, Y)` for all variations
- 

![image](https://github.com/user-attachments/assets/ea3a9689-163d-4c64-85a1-2fdd8d1c1b43)

Reply:
I'd just fix configs and add safeguards, at least for now.
In future, I'd like to implement 3x4 matrix for all sensorts, so scaling, rotation and offset compensation is lumped into single matrix multiplication.

'standard' alignments (n*90 deg, flip) is easier to understand and enoung in most cases, RPY is harder to understand and dependent on order of operations (and it not unique - there are multiple combinations to get identical orientation). We can use if internally (mt PR has macro to do the conversion), but  standard orientation is simpler for config

Reply:
@ledvinap please re-approve if this is ready for now